crates/sui-package-management/Cargo.toml

```diff
@@ -0,0 +1,30 @@
+[package]
+name = "sui-package-management"
+version.workspace = true
+authors = ["Mysten Labs <build@mystenlabs.com>"]
+license = "Apache-2.0"
+publish = false
+edition = "2021"
+
+[lib]
+path = "src/lib.rs"
+
+[dependencies]
+anyhow.workspace = true
+tracing.workspace = true
+
+sui-json-rpc-types.workspace = true
+sui-sdk.workspace = true
+
+move-package.workspace = true
+
+[dev-dependencies]
+
+#expect-test.workspace = true
+#rand.workspace = true
+#tempfile.workspace = true
+#tokio = { workspace = true, features = ["macros", "test-util"] }
+#tracing.workspace = true
+
+#sui-test-transaction-builder.workspace = true
+#test-cluster.workspace = true
```

Comment:
nit: remove?

Reply:
whoops! missed staging changes that gets rid of this